genesis/engine/solvers/rigid/gjk_decomp.py

```diff
@@ -0,0 +1,3394 @@
+import taichi as ti
+import genesis as gs
+import genesis.utils.geom as gu
+from .support_decomp import Support
+
+EPA_P2_NONCONVEX = 2
+EPA_P2_FALLBACK3 = 3
+EPA_P3_BAD_NORMAL = 4
+EPA_P3_INVALID_V4 = 5
+EPA_P3_INVALID_V5 = 6
+EPA_P3_MISSING_ORIGIN = 7
+EPA_P3_ORIGIN_ON_FACE = 8
+EPA_P4_MISSING_ORIGIN = 9
+EPA_P4_FALLBACK3 = 10
+
+
+@ti.data_oriented
+class GJK:
+    def __init__(self, rigid_solver):
+        self._solver = rigid_solver
+        self._B = rigid_solver._B
+
+        self.FLOAT_MIN = gs.np_float(1e-15)
+        self.FLOAT_MIN_SQ = self.FLOAT_MIN * self.FLOAT_MIN
+        self.FLOAT_MAX = gs.np_float(1e15)
+        self.FLOAT_MAX_SQ = self.FLOAT_MAX * self.FLOAT_MAX
+        self.tolerance = gs.np_float(1e-6)
+        self.collision_eps = gs.np_float(1e-6)
+
+        # Maximum number of contact points to find per pair
+        self.max_contacts_per_pair = 50
+
+        ### GJK
+        self.gjk_max_iterations = 50
```

Comment:
Can you group hyper-parameters like this together on top of the init to distinguish them from the rest of the code easily?